e2e/scenario_win_test.go

```diff
@@ -28,7 +36,10 @@ func Test_Windows2022Containerd(t *testing.T) {
 			VHD:             config.VHDWindows2022Containerd,
 			VMConfigMutator: func(vmss *armcompute.VirtualMachineScaleSet) {},
 			BootstrapConfigMutator: func(configuration *datamodel.NodeBootstrappingConfiguration) {
-
+			},
+			Validator: func(ctx context.Context, s *Scenario) {
+				ValidateFileHasContentWindows(ctx, s, "/k/kubeletstart.ps1", "--container-runtime=remote")
+				//ValidateProcessHasCliWindows(ctx, s, "kubelet.exe", []string{"--address=0.0.0.0", "--anonymous-auth=false"})
```

Comment:
can we remove the commented-out bits?

Reply:
I'm working on a PR to get those validations working - that I'm going to include in this PR due to some of the other comments. So I'll resolve this comment by getting those working.